engine/src/gfx/jpeg_memory.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
The generally accepted method in the project is to use guards instead of this pragma:
```
#ifndef _FILENAME_H
#define _FILENAME_H

// File content

#endif
```

Reply:
Ah, this was my IDE being crazy again. I've removed it now.

Reply:
Yeah, there's two methods of header guards:
```
#pragma once
```
where the compiler does all the work
and
```
#ifndef __SOME_CONSTANT
#define __SOME_CONSTANT
...
#endif //__SOME_CONSTANT
```
where the programmer does more of the work.

Microsoft and some communities prefer the `pragma once` method; however, I've found in practice that the `ifndef/define/endif` method is far easier to trace, debug, etc when things go wrong. So I agree with @nabaco that that should be our preferred method. We can update the style-guide if needed.